bulk_writer.js

```diff
@@ -61,12 +61,17 @@ BulkWriter.prototype.flush = function flush() {
   const bulk = this.bulk.concat();
   this.bulk = [];
   debug('going to write', bulk);
-  return this.client.bulk({
+  var bulkParams = {
     body: bulk,
     consistency: this.consistency,
     timeout: this.interval + 'ms',
     type: this.type
-  }).catch((e) => {
+  };
+
+  if (!this.consistency) {
```

Comment:
we can do this in the return:

 `consistency: (this.consistency !== false) : this.consistency : undefined`